Sources/AblyChat/Timeserial.swift

```diff
@@ -0,0 +1,101 @@
+import Foundation
+
+// Timeserial Protocol
+protocol Timeserial: Sendable {
+    var seriesId: String { get }
+    var timestamp: Int { get }
+    var counter: Int { get }
+    var index: Int? { get }
+
+    func toString() -> String
+    func before(_ timeserial: Timeserial) -> Bool
+    func after(_ timeserial: Timeserial) -> Bool
+    func equal(_ timeserial: Timeserial) -> Bool
+}
+
+// DefaultTimeserial Class
+final class DefaultTimeserial: Timeserial {
```

Comment:
Why a class? Looks like something struct-like to me. Does it need reference semantics?

Reply:
you're right, fixed